src/open-data/Services.csv

```diff
@@ -1,314 +1,314 @@
+Microsoft.ElasticSan;microsoft.elasticsan/elasticsans;Azure Elastic SAN;Storage;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.EventGrid;microsoft.eventgrid/domains;Event Grid;Integration;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventGrid;microsoft.eventgrid/extensiontopics;Event Grid;Integration;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventGrid;microsoft.eventgrid/namespaces;Event Grid;Integration;Microsoft;Cloud Provider;Cloud;PaaS
+MICROSOFT.EVENTGRID;microsoft.eventgrid/partnertopics;Event Grid;Integration;Microsoft;Cloud Provider;Cloud;PaaS
+MICROSOFT.EVENTGRID;microsoft.eventgrid/systemtopics;Event Grid;Integration;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventGrid;microsoft.eventgrid/topics;Event Grid;Integration;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventHub;microsoft.eventhub/clusters;Event Hubs;Integration;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventGrid;microsoft.eventhub/namespaces;Event Hubs;Integration;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventHub;microsoft.eventhub/namespaces;Event Hubs;Integration;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.Fabric;microsoft.fabric/capacities;Microsoft.Fabric;Analytics;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.FluidRelay;microsoft.fluidrelay/fluidrelayservers;Azure Fluid Relay;Web;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.Genomics;microsoft.genomics/accounts;Microsoft Genomics;AI and Machine Learning;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.Graph;microsoft.graph/azureadapplications;Microsoft Graph Data Connect;Integration;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.GraphServices;microsoft.graphservices/accounts;Microsoft Graph;Integration;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.HanaOnAzure;microsoft.hanaonazure/hanainstances;SAP on Azure;Compute;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.HardwareSecurityModules;microsoft.hardwaresecuritymodules/dedicatedhsms;Azure Dedicated HSM;Security;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.HDInsight;microsoft.hdinsight/clusters;HDInsight;Analytics;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.HDInsightFairfax;microsoft.hdinsightfairfax/clusters;HDInsight;Analytics;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.HealthBot;microsoft.healthbot/healthbots;Health Bot;AI and Machine Learning;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.HealthcareApis;microsoft.healthcareapis/services;Azure Health Data Services;Integration;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.EventGrid;microsoft.healthcareapis/workspaces;Azure Health Data Services;Integration;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.HealthcareApis;microsoft.healthcareapis/workspaces;Azure Health Data Services;Integration;Microsoft;Cloud Provider;Cloud;SaaS
+microsoft.hybridcompute;microsoft.hybridcompute/machines;Azure Arc;Multicloud;Microsoft;Cloud Provider;Cloud;SaaS
+;microsoft.hybridnetwork/devices;Azure Network Function Manager;Networking;Microsoft;Cloud Provider;Cloud;IaaS

```

Comment:
I don't knkow anything about this, but should it be "Hybrid", if we used that above? Cloud might be better. Not sure. Mentioning it because of the name.

Reply:
Changed to Hybrid